custom-targets/git-ops/README.md

```diff
@@ -0,0 +1,81 @@
+# Cloud Deploy Git Deployer Sample
```

Comment:
Add overview to give context:

## Overview
The GitOps deployer allows you to use Cloud Deploy to manage your delivery pipeline while using a Kubernetes git synchronization tool (such as Argo) to actually apply changes to the cluster. This enables normal operation and use of Cloud Deploy features (such as progressions, verification, automation, etc…) with the difference being the *deploy job* writes the manifest to a git repository instead of applying to the cluster directly. From there, it is expected that a git syncing tool is running on the clusters which will synchronize its state.

Example use cases:
* You have an existing GitOps setup and would like to extend it with Cloud Deploy features such  as managing the progression sequence, running pre-post deployment actions and verifications
* You have an existing Cloud Deploy pipeline and would like to add git history of all changes to the target
* You want Cloud Deploy to act as your single control pane for all deployment activities and want to connect your GitOps flows into it.